src/components/divider/Divider.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactElement } from "react";
+import { Box, useTheme } from "@adaptui/react-native-tailwind";
+
+import { createComponent, cx } from "../../utils";
+import { Button, ButtonProps } from "../button";
+
+interface DividerProps {
+  label?: string;
+  orientation?: "horizontal" | "vertical";
+  slot?: ReactElement;
+  labelPosition: "start" | "center" | "end";
+  buttonProps: ButtonProps;
+}
+
+const RNDivider = ({
+  label,
+  orientation = "horizontal",
+  slot,
+  labelPosition = "start",
+  buttonProps,
+}: Partial<DividerProps>) => {
+  const tailwind = useTheme();
+  const dividerTheme = useTheme("divider");
+  return (
+    <Box
+      style={tailwind.style(
+        cx(dividerTheme.orientation[orientation]),
+        orientation === "vertical" ? "overflow-hidden" : {},
+      )}
+    >
+      <Box style={tailwind.style(cx(dividerTheme.lines[orientation]))} />
```

Comment:
I think there should be a way to style the line, to handle one of the cases users might have.

Reply:
Added an additional prop to style the divider line